lib/section-id.ts

```diff
@@ -0,0 +1,4 @@
+const getSectionId = (title: string, idSeed: string) =>
+  `${idSeed}-${title.toLowerCase().replace(/[^a-z0-9]/gi, '-')}`;
```

Comment:
@bhgeorge these IDs will change over time. I don't think we should be using `idSeed` in this circumstance. I think we want persistence here. We also won't ever have multiple `InPageNav` component on a page, I think it's fine to ditch it entirely. That way the IDs are consistent until content changes.

Reply:
Sounds good was back and forth on that one